src/Servers.cpp

```diff
@@ -1826,7 +1826,16 @@ void Server::rpc_blockchain_transaction_broadcast(Client *c, const RPC::BatchId
     // -- filter may end up applying murmur3 hash to the entire key we give it from 20-50 times!
     const QByteArray txkey = (!rawtxhex.isEmpty() ? BTC::HashOnce(Util::ParseHexFast(rawtxhex)).left(16) : QByteArrayLiteral("xx"));
     // no need to validate hex here -- bitcoind does validation for us!
-    generic_async_to_bitcoind(c, batchId, m.id, "sendrawtransaction", QVariantList{ rawtxhex },
+    QVariantList params { rawtxhex } ;
+    if ((bitcoindmgr->isCoreLike())
+            && bitcoindmgr->getBitcoinDVersion() >= Version{0,25,0}) {
```

Comment:
Shouldn't version be `Version{25,0,0}` ? Didn't they redo their version scheme to semver?

Reply:
I was confused by this as well.  While the string version changed, the integer version remained consistent with pre-22.0 releases (e.g. getnetworkinfo returns 250000 for 25.0).  So in Server.h the result of version = Version::BitcoinDCompact(val) is 0.25.0

Reply:
Jesus wept. Oh Core….

so you checked and are sure? Ok. Lord… :)

Reply:
:-) Indeed.
Double checked to be sure.  While the string "subversion" output of getnetworkinfo is "/Satoshi:25.0.0/", the integer "version" is 250000.  It seems they just started treating minor as major, and revision as minor in order to effect the switch.  I don't see why they did it this way either because keeping the integer format the same and just going to major version 22 wouldn't have broken anyone relying on the raw integer comparison either

pre-22.0 in bitcoin/src/clientversion.h:
```
static const int CLIENT_VERSION =
                           1000000 * CLIENT_VERSION_MAJOR
                         +   10000 * CLIENT_VERSION_MINOR
                         +     100 * CLIENT_VERSION_REVISION
                         +       1 * CLIENT_VERSION_BUILD;
```
22.0 and up:
```
static const int CLIENT_VERSION =
                             10000 * CLIENT_VERSION_MAJOR
                         +     100 * CLIENT_VERSION_MINOR
                         +       1 * CLIENT_VERSION_BUILD;
```
I suppose one way to avoid the confusion is to always parse the string out of subversion, but I'm unsure if this will break anything for older versions or other protocols